GVFS/GVFS.UnitTests/Mock/Virtualization/Projection/MockGitIndexProjection.cs

```diff
@@ -67,6 +67,11 @@ public override bool IsProjectionParseComplete()
             return this.ProjectionParseComplete;
         }
 
+        public override PathProjectionState GetPathProjectionState(string virtualPath)
+        {
+            return PathProjectionState.Included;

```

Comment:
Similar comment to the comment in `SortedFolderEntriesTests.cs`.  Are there any unit tests that should be run with `GetPathProjectionState` returning something other than `Included`?

Reply:
Look into this and it was a lot more work than I thought so I think I will add an issue.  The `MockGitIndexProjection` is kind of a half mock in that it overrides some methods but derives from `GitIndexProjection` but calls a constructor that doesn't setup anything so I was getting a bunch of `NullReferenceException`s and that led me down a rabbit hole.

Reply:
Added issue #1366